.github/workflows/runnable_cxx.yml

```diff
@@ -287,6 +287,16 @@ jobs:
           ./dmd/test/run.d runnable_cxx MODEL=32
         fi
 
+    ########################################
+    #      Run C++ frontend unittests      #
+    ########################################
+    - name: Run C++ frontend unittests
+      run: |
+        ./dmd/src/build.d cxx-unittest MODEL=64
```

Comment:
```suggestion
        ./dmd/src/build.d -j2 cxx-unittest MODEL=64
```

Ditto below

Reply:
I don't think any part of it can be parallelized...

Reply:
Okay. Note that parallelization is enabled by default, this just limits the maximum job count.

Reply:
Ah, well the pipeline is sequential, so no risk of eating up our quota of cores here.